src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,192 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
+  const [productFormBrand, setProductFormBrand] = useState("");
+  const [productFormTitle, setProductFormTitle] = useState("");
+  const [productFormDescription, setProductFormDescription] = useState("");
+  const [productFormCategory, setProductFormCategory] = useState("");
+  const [productFormPrice, setProductFormPrice] = useState("");
+
+  // Add products to the database
+  const addNewProductToDatabase = async (e) => {
+    e.preventDefault();
+
+    // Getting all images from the inputs and then pushing them into an array to post the data to the database
+    const allImages = [];
+    const imageInputs = document.getElementsByTagName("input");
+    for (let i = 0; i < imageInputs.length; i++) {
+      if (imageInputs[i].placeholder === "Image") {
```

Comment:
checking the placeholder like this is also not a good practice. imagine your teammate 2 weeks down the line wants to change the placeholder text, but they don't know that this code uses the placeholder to hardcode something (nor should they expect it).

what you want to do here is add onchange handler to the inputs to modify state, just as you would with any other form item